src/users/users.controller.ts

```diff
@@ -15,28 +18,35 @@ import {
 } from './dto/create-user.dto';
 import { UpdateUserDto } from './dto/update-user.dto';
 import { UserObj } from 'src/decorators/userobj.decorator';
+import {
+  FileFieldsInterceptor,
+  FileInterceptor,
+} from '@nestjs/platform-express';
+import { FileImport } from '../types';
+import { dirname } from 'path';
+import { storagePath } from '../config/storage/storage.config';
 
-@Controller('users')
+@Controller('/users')
 export class UsersController {
   constructor(private readonly usersService: UsersService) {}
 
-  @Post()
-  create(@Body() createUserDto: CreateUserDto) {
-    return this.usersService.create(createUserDto);
-  }
+  // @Post()
+  // create(@Body() createUserDto: CreateUserDto) {
+  //   return this.usersService.create(createUserDto);
+  // }
 
   // TODO: wstęp tylko dla admina - zabezpieczyc
-  @Post('hr')
+  @Post('/hr')
```

Comment:
po co jest dodany / praktycznie wszędzie ? nie jest potrzebny

Reply:
Właśnie z tego względu to zmieniłem tak jak napisała joanbier